pkgs/os-specific/linux/systemd/default.nix

```diff
@@ -24,8 +24,8 @@ stdenv.mkDerivation rec {
   src = fetchFromGitHub {
     owner = "NixOS";
     repo = "systemd";
-    rev = "nixos-v${version}";
-    sha256 = "0ldyhfxdy4qlgygvpc92wp0qp6p1c9y3rnm77zwbkga48x60d9i8";
+    rev = "045ec3f81571b71beebcd2557e1e81e930763d34";
```

Comment:
this should be flipped back to `nixos-v${version}` once https://github.com/NixOS/systemd/pull/29 is merged.

Reply:
flipped back.